server/api.go

```diff
@@ -60,13 +60,13 @@ func (p *Plugin) getGo(w http.ResponseWriter, r *http.Request) {
 	}
 
 	source := r.URL.Query().Get("source")
-	if len(source) < 2 || len(source) > 4 {
+	if len(source) < 2 || len(source) > 5 { 
```

Comment:
Failed on lint due to extra whitespace
```suggestion
	if len(source) < 2 || len(source) > 5 {
```